cli/providers/aws/translate.go

```diff
@@ -114,25 +117,32 @@ sudo docker run \
 	return base64.StdEncoding.EncodeToString([]byte(value))
 }
 
-func loadELB(t *gocf.Template, s *service.Service, e *environment.Environment) {
-	instancePort := ""
+func loadELB(t *gocf.Template, s *service.Service, e *environment.Environment) error {
+	healthCheckPort := ""
 	listeners := gocf.ElasticLoadBalancingListenerList{}
 	for _, container := range s.Containers {
 		for _, port := range container.Ports {
-			if instancePort == "" {
-				instancePort = port.InstancePort
+			if healthCheckPort == "" {
+				healthCheckPort = port.InstancePort
+			}
+			certificate := port.Certificate
```

Comment:
What happens if the port doesn't require SSL?

Reply:
I will add a checking for that case, thanks!